environment.yml

```diff
@@ -3,7 +3,6 @@ channels:
     - conda-forge
     - defaults
 dependencies:
-    - python=3.8
```

Comment:
What about adding a few comment lines to group the dependencies into:

1. required dependencies
2. dev dependencies
3. CI-specific dependencies (codecov, and make?)

Reply:
Ok. But should we consider `make` a CI dependency, or a dev dependency? I feel as though `make` here might help on Windows, see https://github.com/GenericMappingTools/pygmt/issues/914#issuecomment-780123464.

Reply:
Perhaps just do required and dev dependencies (see the Suggested Changes I made below). [`codecov`](https://pypi.org/project/codecov/2.1.11/#files) is a tiny 16.4kb wheel. [`make`](https://anaconda.org/conda-forge/make/files) is 6.0MB on Windows but could be considered a dev dependency too.

Reply:
Looks good.